testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketReadPendingTest.java

```diff
@@ -40,7 +40,7 @@
 import static org.junit.Assert.assertTrue;
 
 public class SocketReadPendingTest extends AbstractSocketTest {
-    @Test(timeout = 30000)
+    @Test(timeout = 35000)
```

Comment:
Was some timeout that occurred per permutation, had to increase this to account for the new permutation.